asv_bench/benchmarks/join_merge.py

```diff
@@ -1,10 +1,6 @@
 from .pandas_vb_common import *
 
-try:
-    from pandas import merge_ordered
-except ImportError:
-    from pandas import ordered_merge as merge_ordered
```

Comment:
You can still leave this one here (in case you want to compare a benchmark with pandas 0.18, which is quite unlikely, so we need to decide at a certain point how to deal with such things in the asv benchmarks, but let's leave that for another issue)

Reply:
I'm not sure I understand: ``merge_ordered`` will always exist, so the ImportError is never reached and hence the try/except can be removed. Or am I misunderstanding something?

Reply:
Eg in pandas 0.18.0 `merge_ordered` does not exist, and then the except part will be reached. The thing with benchmarks is that you run the latest (master) version of the benchmark also on older code (so benchmarks in master do not only need to satisfy master itself)

Reply:
Ok, I understand, thansk for explaining. I will update the PR later tonight.